app/services/session_encryptor.rb

```diff
@@ -1,4 +1,6 @@
 class SessionEncryptor
+  MARSHAL_SIGNATURE ||= 'BAh'.freeze
```

Comment:
I'm not sure it's worth it since this code is temporary. Once it's been deployed to production, we should stop seeing Marshalled sessions within an hour I would think, and we can then remove this. This will only affect sessions that are in-progress while this code is being deployed. Once those sessions are over, this code won't matter anymore.